Legacy versions of mkdirp are no longer supported.

Cannot install I'm getting an error by npm-

npm WARN deprecated mkdirp@0.5.1: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note
that the API surface has changed to use Promises in 1.x.)
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/notmasteryet/jpgjs.git
npm ERR! enoent
npm ERR! enoent
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:

--- a/index.js
+++ b/index.js
@@ -2,7 +2,7 @@
 const util = require('util');
 const fs = require('fs');
 const chalk = require('chalk');
-const Jimp = require('@sindresorhus/jimp');
+const Jimp = require('jimp');
 const termImg = require('term-img');
 
 const PIXEL = '\u2584';

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
 		"pixels"
 	],
 	"dependencies": {
-		"@sindresorhus/jimp": "^0.3.0",
+		"jimp": "^0.10.3",
 		"chalk": "^4.0.0",
 		"term-img": "^4.1.0"
 	},

